metricbeat/docs/fields.asciidoc

```diff
@@ -2790,28 +2790,28 @@ Bytes in non-idle span.
 
 
 [[exported-fields-graphite]]
-== graphite Fields
+== graphite fields
```

Comment:
@dedemorton Seems like some of your recent changes are revert? I wonder why the previous builds went green?